app/assets/stylesheets/_home_cards.css.scss

```diff
@@ -0,0 +1,75 @@
+/* Home cards */
+.grouped-list-none {
+  padding: 0;
+  text-align: center;
+}
+.list-group-item,
+.list-group-item:hover,
+.list-group-item:focus {
+  -moz-border-radius: 3px;
+  -webkit-border-radius: 3px;
+  border-radius: 3px;
+  -moz-background-clip: padding;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
+  background-color: #fff;
+  -moz-box-shadow: 0 2px 2px rgba(0,0,0,.1);
+  -webkit-box-shadow: 0 2px 2px rgba(0,0,0,.1);
+  box-shadow: 0 2px 2px rgba(0,0,0,.1);
+  border: 0;
+}
+.list-group-item {
+  width: 285px;
+  text-align: left;
+  .group-container-none & {
+    display: inline-block;
+    margin: 0 13px 26px;
+    vertical-align: top;
+  }
+  h4 {
+    font-size: 22px;
+    font-weight: 400;
+    padding-bottom: 8px;
+    padding-top: 15px;
+    a {
+      color: black;
+      &:hover {
+        border-bottom: 1px dotted black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.